caasp-389-ds-image/root/usr/local/bin/entrypoint.sh

```diff
@@ -27,38 +27,69 @@ if [ ! -e "/data/config/setup.inf" ]; then
 
     echo -e "\n>> Creating localhost configuration...\n"
     cat <<EOT >> /data/config/setup.inf
-[General]
-FullMachineName=localhost.${DOMAIN_SUFFIX}
-;SuiteSpotUserID=dirsrv
-;SuiteSpotGroup=dirsrv
-;Should not run as root
-SuiteSpotUserID=root
-;Should not run as root
-SuiteSpotGroup=root
-StrictHostCheck=false
+[general]
+config_version = 2
+;defaults = 999999999
+;full_machine_name = localhost.${DOMAIN_SUFFIX}
+selinux = False
+start = False
+strict_host_checking = False
+systemd = False
+start_server = False
+with_systemd = False
 
 [slapd]
-ServerPort=3389
-;SecurePort=3636
-ServerIdentifier=localhost
-Suffix=${DS_SUFFIX}
-RootDN=${ROOT_DN}
-RootDNPwd=${DS_DM_PASSWORD}
-ds_bename=userRoot
-SlapdConfigForMC=Yes
-UseExistingMC=0
-AddSampleEntries=No
-start_server=0
-with_systemd=0
-;self_sign_cert=True
-
+backup_dir = /data/bak
+;bin_dir = /usr/bin
+;cert_dir = /etc/dirsrv/slapd-localhost
+;config_dir = /etc/dirsrv/slapd-localhost
+;data_dir = /usr/share
+db_dir = /data/db
+;db_home_dir = /data/db
+;group = dirsrv
+;initconfig_dir = /etc/sysconfig
+inst_dir = /data
+instance_name = localhost
+ldif_dir = /data/ldif
+;lib_dir = /usr/lib64
+local_state_dir = /data
+lock_dir = /data/run/lock
+log_dir = /data/logs
+port = 3389
+;prefix = /usr
+root_dn = ${ROOT_DN}
+root_password = ${DS_DM_PASSWORD}
+run_dir = /data/run
+;sbin_dir = /usr/sbin
+;schema_dir = /etc/dirsrv/slapd-localhost/schema
+secure_port = 3636
+;self_sign_cert = True
+;self_sign_cert_valid_months = 24
+;sysconf_dir = /etc
+;tmp_dir = /tmp
+;Should not run as root
+user = root
```

Comment:
Should we remove these commented out settings?

Reply:
I create this from CLI 'dscreate create-template'. I keep these variables, so I know what is default value. It can be removed.